src/main/java/at/hannibal2/skyhanni/data/SackAPI.kt

```diff
@@ -258,7 +258,9 @@ object SackAPI {
         val sackEvent = SackChangeEvent(sackChanges, otherItemsAdded, otherItemsRemoved)
         updateSacks(sackEvent)
         sackEvent.postAndCatch()
-        if (chatConfig.hideSacksChange) {
+        if (chatConfig.hideSacksChange && HypixelData.skyBlockIsland != IslandType.GARDEN) {
+            event.blockedReason = "sacks_change"
+        } else if (HypixelData.skyBlockIsland == IslandType.GARDEN && chatConfig.onlyHideSacksChangeOnGarden) {
```

Comment:
This logic is confusing and can be improved

Reply:
(wrong)

Reply:
? What do you mean by (wrong) nopo

Reply:
it wont hide messages on the garden anymore